packages/turbo-workspaces/src/install.ts

```diff
@@ -68,6 +68,18 @@ export const PACKAGE_MANAGERS: Record<
       semver: ">=2",
     },
   ],
+  bun: [
+    {
+      name: "bun1",
+      template: "bun",
+      command: "bun",
+      installArgs: ["install"],
+      version: "1.x",
```

Comment:
This should be `latest`, it's the registry tag.

```suggestion
      version: "latest",
```

Reply:
`1.x` being a weird thing is because of (old) `corepack` mistakes.